tests/security/ecryptfs/ecryptfs.pm

```diff
@@ -0,0 +1,74 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Test all encrypt ciphers of ecryptfs
+# Maintainer: Starry Wang <starry.wang@suse.com> Ben Chou <bchou@suse.com>
+# Tags: poo#110355
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use Utils::Architectures;
+
+sub ecryptfs_mount {
+    my ($cipher) = @_;
+    my $passwd = 'testpasswd';
+
+    assert_script_run('mkdir .private private');
+    my $script = "mount -t ecryptfs -o key=passphrase:passphrase_passwd=$passwd ./.private ./private | tee /dev/$serialdev";
+    enter_cmd($script);
+    # Select cipher
+    enter_cmd($cipher);
+    # Select key bytes, disable plaintext passthrough, disable filename encryption
+    for (1 .. 3) {
+        send_key('ret');
+    }
+    # Would you like to proceed with the mount
+    enter_cmd('yes');
+    # Would you like to avoid this warning in the future
+    enter_cmd('no');
```

Comment:
@STARRY-S 
 Why not using expect here, but not a big deal, enter_cmd should be a simple way

Reply:
Since `mount -t ecryptfs` is failed in fips mode, we cannot use `assert_script_run` to check its results.